django-verdant/rca_ee/models.py

```diff
@@ -107,10 +107,16 @@ def serve(self, request):
         context['form'] = form
         return render(
             request,
-            self.template,
+            self.get_template(request),
             context
         )
 
+    def get_template(self, request, *args, **kwargs):
+        if request.is_ajax() and request.GET.get('format') == 'enquiry_form':
```

Comment:
Thanks. [`cache: false`](https://github.com/torchbox/verdant-rca/pull/82/files#diff-d9d605721c32898c25af186c72649958R810) adds a timestamp into a URL parameters, so it shouldn't be an issue.

P.S.: Sorry for a late response, I missed a notification about this message past week.